src/components/WorkOrder/RateScheduleItems/AddedRateScheduleItems.js

```diff
@@ -11,9 +11,11 @@ const AddedRateScheduleItems = ({
   sorSearchRequest,
   sorCodeArrays,
   setSorCodeArrays,
+  setCurrentPage,
 }) => {
   const [rateScheduleItems, setRateScheduleItems] = useState([...addedTasks])
   const [nextFreeIndex, setNextFreeIndex] = useState(addedTasks.length)
+  const ADDING_MULTIPLE_SOR_PAGE = 2
```

Comment:
See other suggestion about this const duplication - better to pass down a specific function prop here which manages the page change instead, without this component knowing about how the parent keeps track of which page to show.